packages/terra-date-picker/src/DateUtil.jsx

```diff
@@ -0,0 +1,33 @@
+import moment from 'moment';
```

Comment:
should this file be js instead of jsx? Since it doesn't have any react components in it.